geometry/tagged/disjoint.js

```diff
@@ -1,10 +1,11 @@
 import { rewrite, visit } from './visit.js';
 
 import { cut } from './cut.js';
+import { disjoint as disjoinGraphs } from '../graph/disjoint.js';
 import { taggedGroup } from './taggedGroup.js';
 import { toConcreteGeometry } from './toConcreteGeometry.js';
 
-export const disjoint = (geometries) => {
+export const disjoint = (geometries, strategy = 'incremental') => {
```

Comment:
Is incremental the new one?

Reply:
Yeah, but you can switch back if it has problems.

Reply:
Great 👍, I just wanted to make sure I tested the right one 😁